components/scream/cime_config/config_compsets.xml

```diff
@@ -15,12 +15,24 @@
     <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+    <alias>F2010-SCREAM-SA</alias> <!-- SCREAM standalone -->
+    <lname>2010_SCREAM_SLND_SICE_SOCN_SROF_SGLC_SWAV</lname>
+    <support_level>Experimental, under development</support_level>
+  </compset>
+
   <compset>
       <alias>F2000SCREAMv1</alias> <!-- Coupled land-atm compset (using SCREAMv1), 2000 initial conditions -->
       <lname>2000_SCREAM_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV_SIAC_SESP</lname>
       <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+      <alias>F2010SCREAMv1</alias> <!-- Coupled land-atm compset (using SCREAMv1), 2010 initial conditions -->
```

Comment:
We can certainly call these aliases whatever we want, but we have kind of been trying to enforce the convention `<forcing>-<model configuration>-<modifiers>` with EAM and SCREAMv0 compset. So we have things like this: `F2010-SCREAM-HR`. What do you think about renaming these aliases `F2010-SCREAMv1` for consistency with these?

Reply:
Yeah I'd just kept the same syntax as the F2000 compset, but that sounds like a good idea.